#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_default-requantizer.dpatch by Tobias Grimm <tg@e-tobi.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Make transcode's tcrequant the default requantize as long as
## DP: metakine's requantizer is not available for Debian.

@DPATCH@
diff -urNad vdr-plugin-burn-0.0.009+pre21~/setup.c vdr-plugin-burn-0.0.009+pre21/setup.c
--- vdr-plugin-burn-0.0.009+pre21~/setup.c	2006-09-16 20:33:37.000000000 +0200
+++ vdr-plugin-burn-0.0.009+pre21/setup.c	2006-09-23 22:23:20.000000000 +0200
@@ -87,7 +87,7 @@
 			PROCTOOLS_INIT_PROPERTY( CustomDiskSize,      200 ),
 			PROCTOOLS_INIT_PROPERTY( BurnSpeed,           0 ),
 			PROCTOOLS_INIT_PROPERTY( DemuxType,           demuxtype_vdrsync ),
-			PROCTOOLS_INIT_PROPERTY( RequantType,         requanttype_metakine ),
+			PROCTOOLS_INIT_PROPERTY( RequantType,         requanttype_transcode ),
 			PROCTOOLS_INIT_PROPERTY( PreserveLogFiles,    false ),
 			PROCTOOLS_INIT_PROPERTY( DefaultLanguage,     0 ),
 			PROCTOOLS_INIT_PROPERTY( MainMenuStatus,      true ),
